### ISSUE TITLE
[InlineEdit] Unable to edit text field as a prepend

### ISSUE THREAD

**Issue by llupa** [symfony/ux#1490]
Hello,

At https://ux.symfony.com/demos/live-component/inline-edit when positioning your cursors anywhere other than the end of the input field after your first keystroke the cursors jumps at the end. 

I wanted to type `Yellow Banana 🍌` and this is how it looks like on first key-stroke. To achieve a prepend, one has to move the cursor per letter.

<img width="652" alt="Screenshot 2024-02-12 at 11 29 37" src="https://github.com/symfony/ux/assets/41073314/96b9e771-abaa-4e9c-aa71-edc859a24494">

Unfortunately I cannot help with this since I am looking at the demos to get back into trying JS after a decade of not working with it, but I wanted to report the issue at least.

**Comment by globaldigitalagency** [comment]
Hello,

We're also having the same problem in a project.
On component update/render, the text cursor is repositioned at the end of input disregarding its position at the moment of live component update. 
So editing from the middle of the input value or at the beginning of its value cannot be done properly without re-positioning the cursor after each char entered.


We are using these versions in composer.json:
![image](https://github.com/symfony/ux/assets/109951482/ef467c23-1ada-4e52-ace8-2175aad3eb03)

Component is called in a simple controller  rendering template index.html.twig:
```twig
{{ component('Test') }}
```

Component template Test.html.twig:
```twig
<div {{ attributes }}>
  <input type="number" data-model="max">

  Generating a number between 0 and {{ max }}
  <strong>{{ this.randomNumber }}</strong>
</div>
```
Component code:
```php
<?php

namespace App\Twig\Components;

use Symfony\UX\LiveComponent\Attribute\AsLiveComponent;
use Symfony\UX\LiveComponent\Attribute\LiveProp;
use Symfony\UX\LiveComponent\DefaultActionTrait;

#[AsLiveComponent]
class Test
{
    use DefaultActionTrait;

    #[LiveProp(writable: true)]
    public int $max = 1000;

    public function getRandomNumber(): int
    {
        return rand(0, $this->max);
    }
}
```

**Comment by weaverryan** [comment]
Fixed and test case added over at #1501 - we expected a few hiccups when we converted the internal morphing library from morphdom to idiomorph.
